rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -487,6 +490,60 @@ int __pal_exec(char *path, char *argv[], pal_stdio_fds *stdio, int *exit_code)
 	return 0;
 }
 
+int __pal_create_process(pal_create_process_args *args)
+{
+	int pid;
+	if ((pid = fork()) < 0)
+		return -1;
+	else if (pid == 0) {
+		int exit_code, ret;
+
+		ret = __pal_exec_v1(args->path, args->argv, args->stdio, &exit_code);
```

Comment:
__pal_exec_v1 actually does what a __pal_exec() does, and __pal_exec_v2 doesn't. So don't name __pal_exec_v1, just call it __pal_exec. __pal_exec_v2 is re-used by v2 and v3 but it is not worthy being called __pal_exec_v2. Rename __pal_exec_v2 as wait4child().